test/unit/preset/test_preset.py

```diff
@@ -350,3 +351,19 @@ def test_delete_preset_write_failure(tmp_path: Path) -> None:
         preset_manager.delete_preset(1)
     assert len(preset_manager.presets) == 2
     remove(preset_path)
+
+
+def test_migrate_preset(tmp_path: Path) -> None:
```

Comment:
他にもマイグレーションが発生するかもなので、何をテストしたいかの意図をちょっとだけ含めるとさらに良いかも。
こんな感じでしょうか。

```suggestion
def test_migrate_default_preset_path(tmp_path: Path) -> None:
```